lib/friends.rb

```diff
@@ -3,5 +3,5 @@
 require "friends/version"
 
 module Friends
-  TAG_REGEX = /@\p{Alnum}+/
+  TAG_REGEX = /(?<!\p{Alnum})@\p{Alnum}(?:[:-]?\p{Alnum})*/
```

Comment:
Ah, smart move with the negative lookbehind at the beginning. Technically this could inadvertently catch some false positives like very weird emails (`example.@example.com`) but that's so edge-casey I don't think we should worry about it. It's definitely better than before!